### DISCUSSION TITLE
Failed to connect to the host via ssh: ssh: connect to host

### DISCUSSION THREAD

**Discussion by vravru135** [learndevopsonline/learndevopsonline#1678] [Q&A] [answered]
![IMG20231216213720](https://github.com/learndevopsonline/learndevopsonline/assets/151849773/2e757161-b838-4d58-af21-62557db51a76)

**Comment by vravru135** [comment]
![IMG20231215130720](https://github.com/learndevopsonline/learndevopsonline/assets/151849773/25929aae-e0dc-4ecd-ac2d-b28e86e4ee02)

**Reply by vravru135** [reply]
i have gave all private IPs to DNS route53...still it's showing like this

**Reply by sreedharm07** [reply]
did the error still exists ?

**Reply by vravru135** [reply]
yes ...still it is showing like this....pls help me

**Reply by sreedharm07** [reply]
ping your ip

**Reply by sreedharm07** [reply]
your sql server is not able to ssh 
check sql servers security groups and check mysql server once by loging in into your sql server from putty if its loging in it will work in ansible.
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/bff2a04b-be3c-4f11-bdbb-86b95865e599)

**Comment by vravru135** [comment]
![Screenshot (1)](https://github.com/learndevopsonline/learndevopsonline/assets/151849773/a353d1bf-eb2e-4e10-bdd1-d0eb1c2e55f0)

still failing sir .....

**Comment by vravru135** [comment]
![Screenshot (2)](https://github.com/learndevopsonline/learndevopsonline/assets/151849773/a37e60a3-ce7a-4dd5-932d-623b9ce37230)
  
   I have edited my security groups of mysql  too......

**Reply by sreedharm07** [reply]
just try to ssh mysql server manually 
and please reply here weather it is able to login or not
because its not able to ssh for me

**Reply by vravru135** [reply]
sir , can u elaborate the steps...i'm getting u ..sorry

**Reply by sreedharm07** [reply]
in aws instance open mysql instance 
take that public ip and log in into putty and see is it able to log in or not.

**Reply by vravru135** [reply]
it is logging in

**Reply by vravru135** [reply]
sir ..if it is possible ..can i share my screen on google meet

**Reply by sreedharm07** [reply]
join here
https://us06web.zoom.us/j/88648359147?pwd=5acbvxGj6oNc5wJnBCupSQAZ0HrWRI.1#success

**Reply by vravru135** [reply]
still showing same failure sir

**Comment by vravru135** [comment]
![Uploading Screenshot (4).png…]()

**Reply by sreedharm07** [reply]
ping all your ips here 
workstation frontend backend and mysql public ips

**Reply by vravru135** [reply]
public ip
frontend: 54.224.229.85
mysql: 35.175.251.243
backend: 3.81.207.100
workstation: 54.147.253.123

**Comment by vravru135** [comment]
ok..sir

**Reply by sreedharm07** [reply]
use only one thread you are opening many threads 
just reply in same  comment dont create new comment

**Reply by vravru135** [reply]
![Screenshot (5)](https://github.com/learndevopsonline/learndevopsonline/assets/151849773/b8eaa09e-489a-4a72-a2c3-cde7004e6f6e)

sir , i think my route53 has issue ....can u have a look on it sir..pls

**Reply by sreedharm07** [reply]
yes your dns was not availale in all locations

**Reply by vravru135** [reply]
how to resole it sir ...?

**Reply by sreedharm07** [reply, marked as answer]
just remove hosted zone and configure again 
and it will take 24 hrs to come up until that practice with ips

**Reply by vravru135** [reply]
ok sir ...I'll do...thank u
